parent/pom.xml

```
@@ -301,17 +277,15 @@
                         </executions>
                     </plugin>
                     <plugin>
-                        <groupId>org.sonatype.plugins</groupId>
-                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <groupId>org.sonatype.central</groupId>
+                        <artifactId>central-publishing-maven-plugin</artifactId>
                         <extensions>true</extensions>
```

```suggestion
                        <extensions>true</extensions>
                        <configuration>
                            <autoPublish>${autoPublish}</autoPublish>
                        </configuration>
```
suggestion: make autoPublish configurable